frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart

```diff
@@ -98,6 +98,9 @@ class CalendarDayCard extends StatelessWidget {
   }
 
   List<GestureDetector> _buildCards(BuildContext context) {
+    events.sort(
```

Comment:
This sorts the events every time the build method is called, no? If you convert this to a stateful widget, then you can sort the events in `initState()`.

Reply:
This was just a fly-by comment. I realized that converting to a stateful widget might not be the most appropriate solution. My only concern is that sorting can get expensive if we sort the events on every build.

Reply:
I moved it to upper level that construct this widget.